### ISSUE TITLE
Add docs for adding an existing web or native project to a new workspace

### ISSUE THREAD

**Issue by ericnakagawa** [ericvicenti/create-react-workspace#3]
Existing projects with disparate implementations of web and native would benefit by being added to a workspace. Let's document how to do this.

**Comment by ericvicenti** [comment]
Good idea! Right now we'd need to manually add the new package to `lerna.json`, but ideally we could have a nicer way to automate it. Perhaps by scanning for new modules when we `run bootstrap`
